detox/src/devices/drivers/SimulatorDriver.js

```diff
@@ -127,6 +129,13 @@ class SimulatorDriver extends IosDriver {
   async waitForBackground() {
     return await this.client.waitForBackground();
   }
+
+  _transformLaunchArgs(launchArgs) {
```

Comment:
What is the reason for switching to `_transformLaunchArgs` instead of using `defaultLaunchArgsPrefix` per platform?
How does it work with Android Instrumentation which needs to have `-e` before an argument?